datasets/snow_simplified_japanese_corpus/dataset_infos.json

```diff
@@ -0,0 +1 @@
inal texts are from \"Tanaka Corpus\" (http://www.edrdg.org/wiki/index.php/Tanaka_Corpus).\n", "citation": "@inproceedings{maruyama-yamamoto-2018-simplified,\n    title = \"Simplified Corpus with Core Vocabulary\",\n    author = \"Maruyama, Takumi  and\n      Yamamoto, Kazuhide\",\n    booktitle = \"Proceedings of the Eleventh International Conference on Language Resources and Evaluation ({LREC} 2018)\",\n    month = may,\n    year = \"2018\",\n    address = \"Miyazaki, Japan\",\n    publisher = \"European Language Resources Association (ELRA)\",\n    url = \"https://www.aclweb.org/anthology/L18-1185\",\n}\n\n@inproceedings{yamamoto-2017-simplified-japanese,\n    title = \"\u3084\u3055\u3057\u3044\u2f47\u672c\u8a9e\u5bfe\u8a33\u30b3\u30fc\u30d1\u30b9\u306e\u69cb\u7bc9\",\n    author = \"\u2f2d\u672c \u548c\u82f1  and\n      \u4e38\u2f2d \u62d3\u6d77  and\n      \u2f93\u5f35 \u2eef\u6674  and\n      \u7a32\u5ca1 \u5922\u2f08  and\n      \u2f29\u5ddd \u8000\u2f00\u6717  and\n      \u52dd\u2f65 \u54f2\u5f18  and\n      \u9ad9\u6a4b \u5bdb\u6cbb\",\n    booktitle = \"\u8a00\u8a9e\u51e6\u7406\u5b66\u4f1a\u7b2c23\u56de\u5e74\u6b21\u5927\u4f1a\",\n    month = 3\u6708,\n    year = \"2017\",\n    address = \"\u8328\u57ce, \u65e5\u672c\",\n    publisher = \"\u8a00\u8a9e\u51e6\u7406\u5b66\u4f1a\",\n    url = \"https://www.anlp.jp/proceedings/annual_meeting/2017/pdf_dir/B5-1.pdf\",\n}\n\n@inproceedings{katsuta-yamamoto-2018-crowdsourced,\n    title = \"Crowdsourced Corpus of Sentence Simplification with Core Vocabulary\",\n    author = \"Katsuta, Akihiro  and\n      Yamamoto, Kazuhide\",\n    booktitle = \"Proceedings of the Eleventh International Conference on Language Resources and Evaluation ({LREC} 2018)\",\n    month = may,\n    year = \"2018\",\n    address = \"Miyazaki, Japan\",\n    publisher = \"European Language Resources Association (ELRA)\",\n    url = \"https://www.aclweb.org/anthology/L18-1072\",\n}\n", "homepage": "http://www.jnlp.org/SNOW/T15, http://www.jnlp.org/SNOW/T23", "license": "CC BY 4.0", "features": {"ID": {"dtype": "string", "id": null, "_type": "Value"}, "original_ja": {"dtype": "string", "id": null, "_type": "Value"}, "simplified_ja": {"dtype": "string", "id": null, "_type": "Value"}, "original_en": {"dtype": "string", "id": null, "_type": "Value"}, "proper_noun": {"dtype": "string", "id": null, "_type": "Value"}}, "post_processed": null, "supervised_keys": null, "builder_name": "snow_japanese", "config_name": "snow_t23", "version": {"version_str": "1.1.0", "description": null, "major": 1, "minor": 1, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 6704695, "num_examples": 34300, "dataset_name": "snow_japanese"}}, "download_checksums": {"https://filedn.com/lit4DCIlHwxfS1gj9zcYuDJ/SNOW/T23-2020.1.7.xlsx": {"num_bytes": 3641507, "checksum": "71a8923f024c26c28543663b46aab4e9b6af7c2e8f99a8644190942d8f63c7ca"}}, "download_size": 3641507, "post_processing_size": null, "dataset_size": 6704695, "size_in_bytes": 10346202}}
```

Comment:
```suggestion
{"snow_t15": {"description": "About SNOW T15: The simplified corpus for the Japanese language. The corpus has 50,000 manually simplified and aligned sentences. This corpus contains the original sentences, simplified sentences and English translation of the original sentences. It can be used for automatic text simplification as well as translating simple Japanese into English and vice-versa. The core vocabulary is restricted to 2,000 words where it is selected by accounting for several factors such as meaning preservation, variation, simplicity and the UniDic word segmentation criterion.\nFor details, refer to the explanation page of Japanese simplification (http://www.jnlp.org/research/Japanese_simplification). The original texts are from \"small_parallel_enja: 50k En/Ja Parallel Corpus for Testing SMT Methods\", which is a bilingual corpus for machine translation. About SNOW T23: An expansion corpus of 35,000 sentences rewritten in easy Japanese (simple Japanese vocabulary) based on SNOW T15. The original texts are from \"Tanaka Corpus\" (http://www.edrdg.org/wiki/index.php/Tanaka_Corpus).\n", "citation": "@inproceedings{maruyama-yamamoto-2018-simplified,\n    title = \"Simplified Corpus with Core Vocabulary\",\n    author = \"Maruyama, Takumi  and\n      Yamamoto, Kazuhide\",\n    booktitle = \"Proceedings of the Eleventh International Conference on Language Resources and Evaluation ({LREC} 2018)\",\n    month = may,\n    year = \"2018\",\n    address = \"Miyazaki, Japan\",\n    publisher = \"European Language Resources Association (ELRA)\",\n    url = \"https://www.aclweb.org/anthology/L18-1185\",\n}\n\n@inproceedings{yamamoto-2017-simplified-japanese,\n    title = \"\u3084\u3055\u3057\u3044\u2f47\u672c\u8a9e\u5bfe\u8a33\u30b3\u30fc\u30d1\u30b9\u306e\u69cb\u7bc9\",\n    author = \"\u2f2d\u672c \u548c\u82f1  and\n      \u4e38\u2f2d \u62d3\u6d77  and\n      \u2f93\u5f35 \u2eef\u6674  and\n      \u7a32\u5ca1 \u5922\u2f08  and\n      \u2f29\u5ddd \u8000\u2f00\u6717  and\n      \u52dd\u2f65 \u54f2\u5f18  and\n      \u9ad9\u6a4b \u5bdb\u6cbb\",\n    booktitle = \"\u8a00\u8a9e\u51e6\u7406\u5b66\u4f1a\u7b2c23\u56de\u5e74\u6b21\u5927\u4f1a\",\n    month = 3\u6708,\n    year = \"2017\",\n    address = \"\u8328\u57ce, \u65e5\u672c\",\n    publisher = \"\u8a00\u8a9e\u51e6\u7406\u5b66\u4f1a\",\n    url = \"https://www.anlp.jp/proceedings/annual_meeting/2017/pdf_dir/B5-1.pdf\",\n}\n\n@inproceedings{katsuta-yamamoto-2018-crowdsourced,\n    title = \"Crowdsourced Corpus of Sentence Simplification with Core Vocabulary\",\n    author = \"Katsuta, Akihiro  and\n      Yamamoto, Kazuhide\",\n    booktitle = \"Proceedings of the Eleventh International Conference on Language Resources and Evaluation ({LREC} 2018)\",\n    month = may,\n    year = \"2018\",\n    address = \"Miyazaki, Japan\",\n    publisher = \"European Language Resources Association (ELRA)\",\n    url = \"https://www.aclweb.org/anthology/L18-1072\",\n}\n", "homepage": "http://www.jnlp.org/SNOW/T15, http://www.jnlp.org/SNOW/T23", "license": "CC BY 4.0", "features": {"ID": {"dtype": "string", "id": null, "_type": "Value"}, "original_ja": {"dtype": "string", "id": null, "_type": "Value"}, "simplified_ja": {"dtype": "string", "id": null, "_type": "Value"}, "original_en": {"dtype": "string", "id": null, "_type": "Value"}}, "post_processed": null, "supervised_keys": null, "builder_name": "snow_simplified_japanese_corpus", "config_name": "snow_t15", "version": {"version_str": "1.1.0", "description": null, "major": 1, "minor": 1, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 7218115, "num_examples": 50000, "dataset_name": "snow_simplified_japanese_corpus"}}, "download_checksums": {"https://filedn.com/lit4DCIlHwxfS1gj9zcYuDJ/SNOW/T15-2020.1.7.xlsx": {"num_bytes": 3634132, "checksum": "46f6e788e744876b08ca432c585e2942ac6a8a35c8c9ab6687ed7e371a0bc0ac"}}, "download_size": 3634132, "post_processing_size": null, "dataset_size": 7218115, "size_in_bytes": 10852247}, "snow_t23": {"description": "About SNOW T15: The simplified corpus for the Japanese language. The corpus has 50,000 manually simplified and aligned sentences. This corpus contains the original sentences, simplified sentences and English translation of the original sentences. It can be used for automatic text simplification as well as translating simple Japanese into English and vice-versa. The core vocabulary is restricted to 2,000 words where it is selected by accounting for several factors such as meaning preservation, variation, simplicity and the UniDic word segmentation criterion.\nFor details, refer to the explanation page of Japanese simplification (http://www.jnlp.org/research/Japanese_simplification). The original texts are from \"small_parallel_enja: 50k En/Ja Parallel Corpus for Testing SMT Methods\", which is a bilingual corpus for machine translation. About SNOW T23: An expansion corpus of 35,000 sentences rewritten in easy Japanese (simple Japanese vocabulary) based on SNOW T15. The original texts are from \"Tanaka Corpus\" (http://www.edrdg.org/wiki/index.php/Tanaka_Corpus).\n", "citation": "@inproceedings{maruyama-yamamoto-2018-simplified,\n    title = \"Simplified Corpus with Core Vocabulary\",\n    author = \"Maruyama, Takumi  and\n      Yamamoto, Kazuhide\",\n    booktitle = \"Proceedings of the Eleventh International Conference on Language Resources and Evaluation ({LREC} 2018)\",\n    month = may,\n    year = \"2018\",\n    address = \"Miyazaki, Japan\",\n    publisher = \"European Language Resources Association (ELRA)\",\n    url = \"https://www.aclweb.org/anthology/L18-1185\",\n}\n\n@inproceedings{yamamoto-2017-simplified-japanese,\n    title = \"\u3084\u3055\u3057\u3044\u2f47\u672c\u8a9e\u5bfe\u8a33\u30b3\u30fc\u30d1\u30b9\u306e\u69cb\u7bc9\",\n    author = \"\u2f2d\u672c \u548c\u82f1  and\n      \u4e38\u2f2d \u62d3\u6d77  and\n      \u2f93\u5f35 \u2eef\u6674  and\n      \u7a32\u5ca1 \u5922\u2f08  and\n      \u2f29\u5ddd \u8000\u2f00\u6717  and\n      \u52dd\u2f65 \u54f2\u5f18  and\n      \u9ad9\u6a4b \u5bdb\u6cbb\",\n    booktitle = \"\u8a00\u8a9e\u51e6\u7406\u5b66\u4f1a\u7b2c23\u56de\u5e74\u6b21\u5927\u4f1a\",\n    month = 3\u6708,\n    year = \"2017\",\n    address = \"\u8328\u57ce, \u65e5\u672c\",\n    publisher = \"\u8a00\u8a9e\u51e6\u7406\u5b66\u4f1a\",\n    url = \"https://www.anlp.jp/proceedings/annual_meeting/2017/pdf_dir/B5-1.pdf\",\n}\n\n@inproceedings{katsuta-yamamoto-2018-crowdsourced,\n    title = \"Crowdsourced Corpus of Sentence Simplification with Core Vocabulary\",\n    author = \"Katsuta, Akihiro  and\n      Yamamoto, Kazuhide\",\n    booktitle = \"Proceedings of the Eleventh International Conference on Language Resources and Evaluation ({LREC} 2018)\",\n    month = may,\n    year = \"2018\",\n    address = \"Miyazaki, Japan\",\n    publisher = \"European Language Resources Association (ELRA)\",\n    url = \"https://www.aclweb.org/anthology/L18-1072\",\n}\n", "homepage": "http://www.jnlp.org/SNOW/T15, http://www.jnlp.org/SNOW/T23", "license": "CC BY 4.0", "features": {"ID": {"dtype": "string", "id": null, "_type": "Value"}, "original_ja": {"dtype": "string", "id": null, "_type": "Value"}, "simplified_ja": {"dtype": "string", "id": null, "_type": "Value"}, "original_en": {"dtype": "string", "id": null, "_type": "Value"}, "proper_noun": {"dtype": "string", "id": null, "_type": "Value"}}, "post_processed": null, "supervised_keys": null, "builder_name": "snow_simplified_japanese_corpus", "config_name": "snow_t23", "version": {"version_str": "1.1.0", "description": null, "major": 1, "minor": 1, "patch": 0}, "splits": {"train": {"name": "train", "num_bytes": 6704695, "num_examples": 34300, "dataset_name": "snow_simplified_japanese_corpus"}}, "download_checksums": {"https://filedn.com/lit4DCIlHwxfS1gj9zcYuDJ/SNOW/T23-2020.1.7.xlsx": {"num_bytes": 3641507, "checksum": "71a8923f024c26c28543663b46aab4e9b6af7c2e8f99a8644190942d8f63c7ca"}}, "download_size": 3641507, "post_processing_size": null, "dataset_size": 6704695, "size_in_bytes": 10346202}}
```

`snow_japanese`-> `snow_simplified_japanese_corpus `